README.md

```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
+20 Billion Swifts will be reserved; these Swifts are not controlled by any central source, but are under the control of the Delegated Nodes. It's the responsibility of the Delegated Nodes to assign these funds on an as-needed basis to Identity Providers, Delegated Nodes, Swift Citizens, or Swift Accounts in an effort to further the success of the Swift Protocol.
 
 ### Funding Proposals
-When a Swift Citizen or Identity Provider requires funds to perform some task they must submit a public Fund Proposal. This proposal will be written in plain text and signed with the requester's private key and then added to the blockchain. Nodes will then have 1 week to vote on the given proposal. Non-votes are counted as Nos. A majority consensus is required for the funds to be transferred.
+When a Swift Citizen or Identity Provider requires funds to perform some task, they must submit a public funding proposal. This proposal will be written in plain text, signed with the requester's private key and then added to the blockchain. Nodes will then have one week to vote on the proposal. Non-votes are counted as "No"s. A majority consensus is required for the funds to be transferred.
 
 ## Consensus Protocol
-A new consensus mechanism is used in the Swift Protocol that allows for extremely fast block times while still being trustable. The Swift Protocol uses a modified version of Delegated Byzantine Fault Tolerance \[6] that is implemented by NEO. Instead of having each block signed by a majority of the nodes, consensus will be decided based on the longest chain.
+A new consensus mechanism is used in the Swift Protocol that allows for extremely fast block times while still being trustable. The Swift Protocol uses a modified version of the Delegated Byzantine Fault Tolerance algorithm implemented by NEO<sup>[[7]](#7)</sup>. Instead of having each block signed by a majority of the nodes, consensus will be decided based on the longest chain.
 
 ### Selecting Forgers
-When Delegated Nodes are elected, they are placed into an ordered list of blocks that are permitted to forge blocks ordered by the amount of votes received. Nodes will then take turns forging blocks round robin style. When it is a nodes turn to create a block they will have a 10 second window to forge a block. If no block is forged during this duration then the next Delegated Node on the list will also have permission to forge a new block. This will be achieved by using a timestamp server \[5].
+When Delegated Nodes are elected, they are placed into a list of blocks that are permitted to forge blocks, ordered by the amount of votes received. Nodes will then take turns forging blocks round-robin. When it's a node's turn to create a block, they will have a ten-second window to forge a block. If no block is forged during this duration, the next Delegated Node on the list will also have permission to forge a new block. This will be achieved by using a timestamp server <sup>[[6]](#6)</sup>.
```

Comment:
I think it's important to clarify that the list is ordered. This makes it clear how all Delegated Nodes are aware of who is allowed to currently forge a block.

Reply:
Me too, just avoid using the word "ordered" twice, if possible.